src/main/kotlin/uk/gov/justice/hmpps/architecture/defineDocumentation.kt

```diff
@@ -66,13 +66,13 @@ fun writeDependency(w: PrintWriter, v: Version) {
   w.print("[${v.application}](${v.applicationUrl})")
 
   w.print("| ")
-  w.print(v.circleciOrbVersion)
+  w.print(v.circleciOrbVersions.joinToString(", "))
```

Comment:
This is the default separator so it's not needed (unless you want to be explicit)

Reply:
TIL, thanks